Program.cs

```diff
@@ -88,10 +84,12 @@ public static void RunSample(IAzure azure)
 
                 //=============================================================
                 // Send a message to queue.
-                Utilities.SendMessageToQueue(keys.PrimaryConnectionString, queueName, "Hello");
+                await Utilities.SendMessageToQueueAsync(keys.PrimaryConnectionString, queueName, "Hello");

```

Comment:
If the goal is to demonstrate using Service Bus, we definitely shouldn't be forcing developers reading the sample to jump to other classes for core operations like sending and receiving.

Reply:
This seems to be caused by some other PRs i mentioned above also have this part. Getting rid of redundant.

Reply:
I think we're missing the overall point of a sample here; we want to demonstrate "here is how you make the library do something and here is how you work with what it returns to you."    The pattern here is "we'll show you how to make the library do something, but if you want to know about the other stuff, you have to jump around and find another file where we show that.

What I'd like to see is locality of information.   The entire operation should be in-line and accessible, so that developers learning from the sample do not have to parse a bunch of infrastructure and jump around.   It's better for a sample to ignore DRY and other practices in favor of being more clearly readable and understandable.